### PR TITLE
NUI-492 Support Windows 10+

### PR DESCRIPTION
**Pull request by DrGirlfriend**
[NUI-492 Support Windows 10+](https://jira.corp.adobe.com/browse/NUI-492)

Updated eslint and fixed complaints
Added code coverage
Updated readme badges
Added Windows to travis
Changed Readme title
Fixed pathing in unit tests (not sure why I didn't use `path.basename()` in the first place)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ language: node_js
 os:
   - linux
   - osx
+  - windows
```
Comment (alexkli) [inline .travis.yml:15]: Regarding node versions below: do we use this library in production code aka actions? If not, we can drop 10.15 and 10.19.

Comment (tmathern) [reply .travis.yml:15]: (I think it's used in prod).

Comment (DrGirlfriend) [reply .travis.yml:15]: @alexkli It's in asset-compute-devtool

Comment (alexkli) [reply .travis.yml:15]: Yes, so that's NOT an action.

**Pull request merged**